pyscraper/get_links_from_ep.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/python
+
+import operator
+from lxml import etree
+from everypolitician import EveryPolitician
+
+
+def output_file(country, legislature, filename):
+    data = EveryPolitician().country(country).legislature(legislature)
+    output_filename = "../members/{0}.xml".format(filename)
+    root = etree.Element('publicwhip')
+
+    sorted_people = sorted(
+        data.popolo().persons,
+        key=operator.attrgetter('name')
+    )
+    for person in sorted_people:
+        parlparse_id = person.identifier_value('parlparse')
+        if parlparse_id is not None:
+            props = {}
+            if person.twitter:
+                props['twitter_username'] = person.twitter
+            if person.facebook:
+                props['facebook_page'] = person.facebook
+
+            if props:
+                info = etree.Element('personinfo')
+                info.set('id', parlparse_id)
+
+                props_sorted = sorted(
```

Comment:
lxml sorts attributes provided in the constructer by name by default, so if you did:
```
if props:
    props['id'] = parlparse_id
    info = etree.Element('personinfo', props)
```
and then you wouldn't need all the rest.